assets/cms/scss/panels/pages/details/_clipboard.scss

```diff
@@ -1,22 +1,32 @@
 .ccm-panel-add-clipboard-block-item {
   border-bottom: 1px solid $gray-200;
-  margin: 20px 0;
-  padding-bottom: 10px;
+  margin: 5px 0;
+  padding-bottom: 5px;
 
   &:first-of-type {
     border-top: 1px solid $gray-200;
-    padding-top: 20px;
+    padding-top: 5px;
   }
 
   &:last-of-type {
     border-bottom: 0;
   }
 
   .block-content {
+    .block {
+      clear: both;
+    }
+
+    .block-name {
+      span.handle {
+        font-size: 0.75rem;
```

Comment:
This looks a little small, the button text on my install is `0.875rem`

Reply:
This is fix on the latest commit.